### PR TITLE
Setup QR code generation, Added form validation for elderly code input, Styled QR scanner page, Dynamic caretaker screen rendering

### PR DESCRIPTION
**Pull request by bweam**
**Changes:**

- Setup QR code generation
- Elderly code input field validation _(temporary codes are "000000" & "AAAAAA")_
- QR Scanner page UI _(todo: add QR validation)_
- Dynamic caretaker screen rendering
- Styled header components for ConnectElderlyScreen & EditCaretakerScreen

---
image

### CODE REVIEW

File: src/internationalization/translations/th.json [hunk 1]
```diff
@@ -135,7 +135,8 @@
     "cameraHelpText": "Place QR code inside the scanning frame",
```
Comment (pointmekin) [inline src/internationalization/translations/th.json]: Don't forget to translate this helper text na.

File: src/screens/ConnectCaretakerScreen.tsx [hunk 2]
```diff
@@ -3,21 +3,28 @@ import { Text, View, Image } from 'native-base';
...
             <View mt="10" flexDir="row" justifyContent="center">
-              <Image source={QRPlaceholder} alt="QR Code" />
+              <QRCode 
+                logo={{uri: base64Logo}}
+                size={225}
+                quietZone={17.5}
+                logoSize={100}
+                value="todo: insert QR code value here" 
+              />
+              {/* <Image source={QRPlaceholder} alt="QR Code" /> */}
```
Comment (pointmekin) [inline src/screens/ConnectCaretakerScreen.tsx]: Do you still plan to use this commented code?

**Review by pointmekin — CHANGES_REQUESTED** [review]

File: src/screens/caretakers/ConnectElderlyScreen.tsx [hunk 3]
```diff
@@ -42,28 +42,22 @@ const ConnectElderlyScreen = () => {
...
+            width="230"
+            alignSelf="center"
+            borderWidth="3"
+            borderColor="#005DB4"
+            justifyContent="center"
+            alignItems="center"></Box>}
+          showMarker={true}
+          containerStyle={{justifyContent: "center", marginBottom: 120}}
+          onRead={(e) => {
+            console.log('success', e)
```
Comment (SPK137) [inline src/screens/caretakers/ConnectElderlyScreen.tsx:57]: Remove console.log?

**Review by SPK137 — CHANGES_REQUESTED** [review]
Remove console.log?

File: src/screens/caretakers/ConnectElderlyScreen.tsx [hunk 4]
```diff
@@ -41,40 +40,24 @@ const ConnectElderlyScreen = () => {
...
```
Comment (SPK137) [inline src/screens/caretakers/ConnectElderlyScreen.tsx:57]: Remove console.log?

**Review by SPK137 — DISMISSED** [review]

**Review by pointmekin — APPROVED** [review]

**Pull request merged**